logparser/logs/micro/logs.py

```diff
@@ -0,0 +1,32 @@
+# Log Parser for RTI Connext.
+#
+#   Copyright 2016 Real-Time Innovations, Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""Create the dictionary for log functions related to RTI Micro.
+
+Functions:
+  + get_regex_list: Get the regular expressions and function list.
+"""
+from __future__ import absolute_import
+import logparser.logs.micro.micro as micro
+
+
+def get_regex_list():
+    """Return the regular expressions and functions list for this module."""
+    regex = []
+    # Parser entity.
+    regex.append([micro.on_micro_error,
+                  r"\[(\d+)\.(\d+)\]ERROR: ModuleID=(\d+)\ Errcode=(\d+)"])
```

Comment:
Don't parse the timestamp here and modify the logparser script to take between 6 and 9 numbers after the dot.